appassembler-maven-plugin/src/main/java/org/codehaus/mojo/appassembler/daemon/yajsw/YetAnotherJavaServiceWrapperDaemonGenerator.java

```diff
@@ -0,0 +1,317 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.codehaus.mojo.appassembler.daemon.yajsw;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.artifact.Artifact;
+import org.apache.maven.artifact.repository.layout.ArtifactRepositoryLayout;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.mojo.appassembler.daemon.DaemonGenerationRequest;
+import org.codehaus.mojo.appassembler.daemon.DaemonGenerator;
+import org.codehaus.mojo.appassembler.daemon.DaemonGeneratorException;
+import org.codehaus.mojo.appassembler.model.Daemon;
+import org.codehaus.mojo.appassembler.util.DependencyFactory;
+import org.codehaus.mojo.appassembler.util.FormattedProperties;
+import org.codehaus.plexus.logging.AbstractLogEnabled;
+import org.codehaus.plexus.util.IOUtil;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * @author Sandra Parsick
+ * @version $Id$
+ * @plexus.component role-hint="yajsw"
+ */
+public class YetAnotherJavaServiceWrapperDaemonGenerator
+    extends AbstractLogEnabled
+    implements DaemonGenerator
+{
+
+    private static List<String> CORE_LIBS = new ArrayList<String>()
+    {
+        {
+            add("commons/commons-cli-1.4.jar");
+            add("commons/commons-collections-3.2.2.jar");
+            add("commons/commons-configuration2-2.3.jar");
+            add("commons/commons-io-2.6.jar");
+            add("commons/commons-lang3-3.8.1.jar");
+            add("commons/commons-lang-2.6.jar");
+            add("commons/commons-logging-1.1.jar");
+            add("commons/commons-text-1.6.jar");
+            add("commons/commons-vfs2-2.2.jar");
+            add("jna/jna-5.3.1.jar");
+            add("jna/jna-platform-5.3.1.jar");
+            add("netty/netty-all-4.1.36.Final.jar");
+            add("yajsw/ahessian.jar");
+        }
+    };
+
+    private static List<String> EXTENDED_LIBS = new ArrayList<String>()
+    {
+        {
+            add("velocity/velocity-1.7.jar");
+            add("yajsw/hessian4.jar");
+            add("groovy/groovy-2.5.7.jar");
+            add("groovy/groovy-patch.jar");
+        }
+    };
+
+    private static List<String> BAT_SCRIPTS = new ArrayList<String>()
+    {
+        {
+              add("installService.bat");
+              add("runConsole.bat");
+              add("startService.bat");
+              add("stopService.bat");
+              add("uninstallService.bat");
+              add("setenv.bat");
+              add("wrapper.bat");
```

Comment:
Cosmetic: I believe these lines are mis-aligned.

Reply:
Thank you. Done.